bitnami/kube-prometheus/Chart.yaml

```
@@ -35,4 +35,4 @@ sources:
   - https://github.com/bitnami/containers/tree/main/bitnami/prometheus
   - https://github.com/bitnami/containers/tree/main/bitnami/alertmanager
   - https://github.com/prometheus-operator/kube-prometheus
-version: 8.4.1
+version: 8.6.0
```

Sorry, we have new changes, can you please update the chart version again?
```suggestion
version: 8.7.0
```